includes/class-webmention-receiver.php

```diff
@@ -146,13 +146,15 @@ public static function register_routes() {
 			'/endpoint',
 			array(
 				array(
-					'methods'  => WP_REST_Server::CREATABLE,
-					'callback' => array( 'Webmention_Receiver', 'post' ),
-					'args'     => self::request_parameters(),
+					'methods'             => WP_REST_Server::CREATABLE,
+					'callback'            => array( 'Webmention_Receiver', 'post' ),
+					'args'                => self::request_parameters(),
+					'permission_callback' => '__return_true',
```

Comment:
why is this required?

Reply:
https://make.wordpress.org/core/2020/07/22/rest-api-changes-in-wordpress-5-5/

It throws a warning if you don't as of 5.5